src/homework/tic_tac_toe/tic_tac_toe.cpp

```diff
@@ -1 +1,69 @@
-//cpp
\ No newline at end of file
+//cpp
+#include"tic_tac_toe.h"
+#include<iostream>
+#include<string>
+
+using std::string;  using std::cout;    using std::cin;
+
+bool TicTacToe::game_over()
+{
+    return check_board_full();
+}
+
+void TicTacToe::start_game(string first_player)
+{
+    player = first_player;
+    clear_board();
+}
+
+void TicTacToe::mark_board(int position)
+{
+    pegs[position - 1] = player;
+    set_next_player();
+}
+
+string TicTacToe::get_player() const
+{
+    return player;
+}
+
+void TicTacToe::display_board() const
+{
+    cout << pegs[0] << " | " << pegs[1] << " | " << pegs[2] << " | " << "\n";
+    cout << pegs[3] << " | " << pegs[4] << " | " << pegs[5] << " | " << "\n";
+    cout << pegs[6] << " | " << pegs[7] << " | " << pegs[8] << " | " << "\n";
+}
+
+void TicTacToe::set_next_player()
+{
+    if (player == "X")
+    {
+        player = "O";
+    }
+    else
+    {
+        player = "X";
+    }
+}
+
+bool check_boad_full()
```

Comment:
This should be:

bool TicTacToe::check_board_full()
{
    for(auto peg: pegs)
    {    
        if(peg == " ")
        {
            return false;
        }
}
    return true;

}